server/jetstream_api.go

```diff
@@ -932,7 +934,6 @@ func (s *Server) processJSAPIRoutedRequests() {
 				}
 				atomic.AddInt64(&js.apiInflight, -1)
 			}
-			queue.recycle(&reqs)
```

Comment:
Do we lose recycling these with PopOne all together?

Reply:
Nope, the `popOne` function manages it when the slice is emptied.